main.js

```diff
@@ -5,4 +5,82 @@ const navToggle = () => {
+    '<img src="images/modinosdesk.png" alt="Offenbach Grand Prix Homepage">',
+
+  ],
+  text: [
+    'A fully online opera-singing competition, where the audience can comment and applaud, and where the jury comments in full view of the audience.  Automated application process, which includes video and biog upload, as well as an automatically generated contract in the end.',
+
+    'Spoiled Eggs was the original template for 2 projects and those two projects were used to promote Spoiled Eggs.  It is a review site for opera, where people can rage click "applause" or "throwing eggs" and of course comment.',
+
+    'A site meant as an extension of Opera Classica Europa`s Inthega classifieds, which is a platform intended on facilitating touring in Germany.',
+
+    "The advertisement and application process for Opera Classica Europa's Opera Studio program, called the Opera Pathway.",
+
+    'A template intended at hosting music competitions, mostly those that were dedicated to a composer.  The Offenbach GP was such a competition later in time.',
+
+    'The advertisement and application process to enter the Callas Prize competition, which was supposed to take place at the Alte Oper Frankfurt for the finals.  Unfortunately cancelled due to COVID.',
+
+    "My personal singing portfolio and also the first site I ever coded.  In it's initial form it was merely a list, but later I created an interesting, albeit somewhat glitchy, navbar system.",
+
+  ],
+
+  liveButton: [
+    '<a href="https://www.offenbachgp.com/" rel=”noopener”>See Live <img src="images/liveIcon.png" alt="Live Icon"></a>',
+    '<a href="https://spoiledeggs.eu5.org/" rel=”noopener”>See Live <img src="images/liveIcon.png" alt="Live Icon"></a>',
+    '<a href="#" rel=”noopener”>See Live <img src="images/liveIcon.png" alt="Live Icon"></a>',
+    '<a href="#" rel=”noopener”>See Live <img src="images/liveIcon.png" alt="Live Icon"></a>',
+    '<a href="#" rel=”noopener”>See Live <img src="images/liveIcon.png" alt="Live Icon"></a>',
+    '<a href="#" rel=”noopener”>See Live <img src="images/liveIcon.png" alt="Live Icon"></a>',
+    '<a href="https://philipmodinos.ueuo.com" rel=”noopener”>See Live <img src="images/liveIcon.png" alt="Live Icon"></a>',
+
+  ],
+
+  sourceButton: [
+    '<a href="#" rel=”noopener”>See Source <img src="images/gitIcon.png" alt="Source Icon"></a>',
+    '<a href="#" rel=”noopener”>See Source <img src="images/gitIcon.png" alt="Source Icon"></a>',
+    '<a href="#" rel=”noopener”>See Source <img src="images/gitIcon.png" alt="Source Icon"></a>',
+    '<a href="#" rel=”noopener”>See Source <img src="images/gitIcon.png" alt="Source Icon"></a>',
+    '<a href="#" rel=”noopener”>See Source <img src="images/gitIcon.png" alt="Source Icon"></a>',
+    '<a href="#" rel=”noopener”>See Source <img src="images/gitIcon.png" alt="Source Icon"></a>',
+    '<a href="#" rel=”noopener”>See Source <img src="images/gitIcon.png" alt="Source Icon"></a>',
+  ],
+
+};
```

Comment:
Even though your approach for storing the project's data is working, does not meet with the project requirements 😥 The idea is to work with an array of objects as shown next:

    const projects = [{
        name: "Submarine game",
        description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
        image: "link_to_image.jpg",
        technologies: ["Phaser3", "JavaScript", "Jest"],
        link: "link_to_live_version.com",
    },  
    {
        name: 'The Second',
        imgUrl: './images/projects/project-detail-img.png',
        techs: ['Github', 'Bootstrap', 'Git'],
        description: 'The Second and only',
    }]

The benefit of working with this approach is that then you can loop through the array and access the properties for each project like this:

    projects[i].name

By modifying your app to work with this approach, you will very likely find a better way to handle the popup buttons that were causing your mentioned Eslint error. You got this 😎🔥

Reply:
Again, because of the fact that I felt like I was left to fend for myself in this instance (coding partner bailed as well), I'd rather not make such a drastic change until after the week is over.  Besides, the code is an array iteration, but rather than taking all the items from a single array, it takes the same index of all the arrays.  The concept is the same, in essence, as it's iterating through array indices.  In further iterations, I will change it around.  But I'd rather not today since I'm already one day behind.

Reply:
I get what you are saying. Sadly, other code reviewers will mention this as well and point out that you have not met the project requirements. Best of luck 👍